Functions/@xASL_GoNoGo/xASL_GoNoGo.m

```diff
@@ -84,7 +84,7 @@ function delete(obj)
                     state = regexprep(state,'\.status$','');
                 end
             else
-                % WARNING: not locked yet
```

Comment:
Are you sure this warning is really meant to be printed for the user, or was this only an internal comment?

Reply:
No clue. This rewrite just seemed better

Reply:
I would revert this one then ;)

Reply:
Do you mind fixing this? I haven't touched this behavior, I only tweaked the warning.

Reply:
OK. Will investigate and fix this.